### ISSUE TITLE
讲解与视频内的标识不符

### ISSUE THREAD

**Issue by wudiudiu07** [udacity/cn-dlnd-issue-reports#9]
## 课程导航+地址
Keras实现卷积神经网络 >>>>Keras中的模型验证

## 具体描述
视频1:00左右开始处，解说【doing by cheecking its accuracy on the validation set】 ，但是视频内显示在test set下方的是check accuracy of the trained model
